examples/analog_hamiltonian_simulation/04_Running_Analog_Hamiltonian_Simulation_with_local_simulator.ipynb

```diff
@@ -0,0 +1,482 @@
+{
```

Comment:
<ul><li>"It is advised" (passive voice) -&gt; "We recommend" (active voice)</li><li>"the QPU" -&gt; "a QPU"</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/225/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>